packages/frontend/src/pages/Landing/MainLanding.tsx

```diff
@@ -57,60 +59,67 @@ const HeaderBar = () => {
   )
 }
 
-export const MainLanding = () => {
+function estimateCountByHundreds(count: number): string {
+  return count < 100
+    ? count.toString()
+    : (Math.round(count / 100) * 100).toLocaleString() + '+'
+}
+
+export default function MainLanding() {
   const navigate = useNavigate()
+
+  const { loading, data } = useQuery(GET_PLUMBER_STATS)
+  const userCount = data?.getPlumberStats.userCount
+  const executionCount = data?.getPlumberStats.executionCount
+
   return (
     <>
       <HeaderBar />
-      <Container maxW="1600px" position="relative">
-        <Center>
+      <Container position="relative" py={0} px={6}>
+        <HStack
+          wrap={{ base: 'wrap', md: 'nowrap' }}
+          gap={{ base: '2rem', md: 0 }}
+        >
           <VStack
-            textAlign={{ base: 'left', md: 'center' }}
-            align={{ base: 'start', md: 'center' }}
+            align="start"
             py={{ base: '0vh', md: '10vh' }}
             w="100%"
             maxW={{ base: 'unset', lg: '35rem' }}
             spacing={8}
           >
-            <Text textStyle="heading">
-              Focus on more
+            <Text
+              textStyle={{ base: 'heading', md: 'subheading', lg: 'heading' }}
+            >
+              Automating
               <br />
-              important work
+              <Skeleton display="inline" isLoaded={!loading}>
+                {estimateCountByHundreds(executionCount)} tasks
```

Comment:
suggest some defensive code here in case the API fails

```suggestion
                {executionCount ? estimateCountByHundreds(executionCount) : ''} tasks
```

Reply:
roger sir